soh/soh/Enhancements/randomizer/option_descriptions.cpp

```diff
@@ -630,20 +630,19 @@ void Settings::CreateOptionDescriptions() {
     mOptionDescriptions[RSK_KAK_100_SKULLS_HINT] = "Talking to the Cursed Resident in the Skultulla House who is saved after 100 tokens will tell you the reward";
     mOptionDescriptions[RSK_MASK_SHOP_HINT] = "Reading the mask shop sign will tell you rewards from showing masks at the Deku Theatre.";
     mOptionDescriptions[RSK_FULL_WALLETS] = "Start with a full wallet. All wallet upgrades come filled with rupees.";
-    mOptionDescriptions[RSK_BOMBCHUS_IN_LOGIC] =
-        "Bombchus are properly considered in logic. Without this setting, any Bombchu requirement"
+    mOptionDescriptions[RSK_BOMBCHU_BAG] =
+        "Bombchu require thier own bag to be found before use. Without this setting, any Bombchu requirement"
```

Comment:
```suggestion
        "Bombchu require their own bag to be found before use. Without this setting, any Bombchu requirement"
```

Reply:
Bombchus should still be plural there as well.